.circleci/config.yml

```diff
@@ -114,20 +102,17 @@ jobs:
             cd build
             make align-service
 
+      # - run-make-docker:
+      #     design: "telescopic_ota"
+
       - run-make-docker:
-          design: "telescopic_ota"
+          design: "test_vga"
 
       # - run-make-docker:
-      #     design: "test_vga"
+      #     design: "current_mirror_ota"
 
       # - run-make-docker:
-      #     design: "switched_capacitor_filter"
-
-      - run-make-docker:
-          design: "current_mirror_ota"
-
-      - run-make-docker:
-          design: "cascode_current_mirror_ota"
+      #     design: "cascode_current_mirror_ota"
```

Comment:
Can you review this list to ensure the commented lines can be completely removed?

I would like to keep build-make-docker run as fast as possible simply to identify any issues with the build-make-docker flow. Optimally one short test with FLAT=0 and one short test with FLAT=1. If there are any long tests here, move it to one of the test-integration runners.

Reply:
Looks like this runs in a reasonable amount of time (3m 10s from start to finish) so we should be good here.

Reply:
Yes they can be removed

> Can you review this list to ensure the commented lines can be completely removed?